cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpMetadataCollector.java

```diff
@@ -159,10 +161,25 @@ private CloudVmMetaDataStatus getCloudVmMetaDataStatus(CloudResource cloudResour
             for (ForwardingRule item : forwardingRuleList.getItems()) {
                 LoadBalancerType itemType = gcpLoadBalancerTypeConverter.getScheme(item.getLoadBalancingScheme()).getCbType();
                 if (names.contains(item.getName()) && loadBalancerTypes.contains(itemType)) {
+                    Map<String, Object> params = new HashMap<>();
+                    List<String> ports = item.getPorts();
+                    if (ports == null || ports.size() != 1) {
+                        LOGGER.warn("Unexpected port count on {}, {}", item.getName(), ports);
+                    }
+                    try {
+                        String backendService = item.getBackendService();
+                        params.put(GcpLoadBalancerMetadataView.getBackendServiceParam(ports.get(0)), backendService);
```

Comment:
You have a null check above that only lists a warning, but here you access `ports` directly. Should there be some logic to handle the case of `ports` being null?

Reply:
IT was being handled with the catch, but better to do it explicitly